pkg/pillar/types/zedroutertypes.go

```diff
@@ -3209,7 +3209,7 @@ func GetIPBroadcast(subnet net.IPNet) net.IP {
 // reflected in the BitMap Size(32 bytes)
 const (
 	BitMapMax       = 255 // with 0 base, its 256
-	MinSubnetSize   = 8   // minimum Subnet Size
+	MinSubnetSize   = 4   // minimum Subnet Size; one IP for app instance
```

Comment:
Should the comment say `; minimum one IP for app instances`?
The assumption here is that the air gap networks will not need default gateway and can use that slot for additional app instances.

Reply:
Sure, at min we have one for app instance. (.0 and .3 are reserved; .1 is the bridgeIPAddr, and .2 is usable).
Maybe I'll write that as a block comment on top.